.environment-scripts/stellar/configuration_variables.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+  
+# NCEPlibs arguments
+NCEPLIBS_PLATFORM=cheyenne
+NCEPLIBS_COMPILER=intel
+
+# ESMF arguments and environment variables
+ESMF_OS=Linux
+ESMF_COMPILER=intel
+ESMF_SITE=default
+ESMF_CC=mpicc
+
+# FMS environment variables
+FMS_CC=mpicc
+FMS_FC=mpif90
+FMS_LDFLAGS=
+FMS_LOG_DRIVER_FLAGS=
+FMS_CPPFLAGS='-Duse_LARGEFILE -DMAXFIELDMETHODS_=500 -DGFS_PHYS'
+FMS_FCFLAGS='-FR -i4 -r8'
+FMS_MAKE_OPTIONS=
+#FMS_BATS=bat
+
+# fv3gfs-fortran arguments
+FV3GFS_PLATFORM=stellar
+
+# Python requirements environment variables
+MPI4PY_CC=mpicc
+MPI4PY_MPICC=mpicc
+
+# Python wrapper environment variables
+WRAPPER_CC='mpicc'
+WRAPPER_LDSHARED='mpicc -shared'
+
+# 'bat' executable destination
+
```

Comment:
nit: could you describe why this is needed more specifically in the comment (since it is not necessary in the Gaea and gnu-docker cases)?

Reply:
Done